x-pack/plugins/actions/server/builtin_action_types/resilient/service.ts

```diff
@@ -5,6 +5,8 @@
  */
 
 import axios from 'axios';
+// Prefer  importing entire lodash library, e.g. import { get } from "lodash"
+// eslint-disable-next-line no-restricted-imports
 import { omitBy, isNil } from 'lodash/fp';
```

Comment:
I think we need an exception for `lodash/fp` as I'm pretty sure you can't access lodash/fp any other way. I've added that import to the shared bundle so we should allow people to use it.

Reply:
Ah i briefly thought about this and then forgot when I did the find and replace. 

@smith can you help me with what the regex would be for this? 
find `// Prefer importing entire lodash library, e.g. import { get } from "lodash"\n// es-lint-disable-next-line no-restricted-imports\nimport(.*)from 'lodash/fp';`
replace `import$1from 'lodash/fp';

?

Reply:
@spalger just updated the rule and removed comments on lodash/fp imports!